### PR TITLE
stepper demo

### PR DESCRIPTION
**Pull request by Chiragh-itpath**

### CODE REVIEW

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/PassForm.cshtml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+﻿@(Html
+    .DevExtreme()
+    .Form<LoginModelRecord>()
+    .OnInitialized("passFormInitialized")
+    .ID("PassForm")
+    .Items(items =>
+    {
+        items.AddCustomFormTextBox(model => model.Password);
+
+        items
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/PassForm.cshtml]: Crete Reusable component for Previous and Next

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/UserForm.cshtml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Html
+    .DevExtreme()
+    .Form<LoginModelRecord>()
+    .OnInitialized("userFormInitialized")
+    .ID("UserForm")
+    .Items(items =>
+    {
+        items.AddCustomFormTextBox(model => model.Username);
+
+        items
```
Comment (ShubhamG2908) [inline src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/UserForm.cshtml]: Crete Reusable component for Previous and Next

**Review by ShubhamG2908 — CHANGES_REQUESTED** [review]
Added changes

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/PassForm.cshtml (hunk 1 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/PassForm.cshtml]: Done

File: src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/UserForm.cshtml (hunk 2 above)
Comment (Chiragh-itpath) [reply src/ATA.Bluebook/ATA.Bluebook.Web/Views/Jobs/Forms/UserForm.cshtml]: Done

**Pull request merged**